python/cudaq/kernel/ast_bridge.py

```diff
@@ -2726,6 +2730,22 @@ def visit_Continue(self, node):
         else:
             cc.ContinueOp([])
 
+    def promote_operand_type(self, ty, operand):
```

Comment:
Can we move this up with the other utility methods on this visitor class?

Reply:
Done.